psycopg_pool/psycopg_pool/pool.py

```diff
@@ -530,11 +530,11 @@ def check_connection(conn: CT) -> None:
         for instance as `!check` callback when a pool is created.
         """
         if conn.autocommit:
-            conn.execute("SELECT 1")
+            conn.execute("--ping")
```

Comment:
Wouldn't it be even better to execute an empty string? Why to give the backend the task of parsing a comment? (`--ping` is not a command: `--` is a comment marker, I find misleading to make it look like a sort of command line option).

AFAICS the result would be the same:

```python
>>> import psycopg
>>> conn = psycopg.connect()
>>> conn.execute("")
<psycopg.Cursor [EMPTY_QUERY] [INTRANS] (database=piro) at 0x7e1de96cb690>

>>> psycopg.connect().execute("select pg_terminate_backend(%s)", [conn.info.backend_pid])

>>> conn.execute("")
OperationalError: consuming input failed: terminating connection due to administrator command
server closed the connection unexpectedly
	This probably means the server terminated abnormally
	before or while processing the request.
```

Reply:
Agree. I read this suggestion when it was first posted, wondered what this magic `--ping` command was that I'd never heard of before, searched around a bit and found nothing, and the penny never dropped that it was a comment.

Executing an empty string seems clearer.

(Aside: I believe `SELECT` is also a valid postgres query, that also returns an empty resultset. Does that involve the query planner too? If not, I marginally prefer this to the empty string query, as it's a little more obvious that it's just an empty SELECT.)

Reply:
Thanks for the review 😊

Truth be told, I copied what the pgx library in go did. 

I'll get to updating the pr with an empty string in the next couple of days.

Reply:
@dvarrazzo @asqui thank you both for the review and sorry it took me that long to do this small change 😅 

I've updated the PR to use an empty string instead of the `--ping` comment _marker_ (thanks for the correction)

Reply:
Thank you! No worries, I will merge this MR as soon as I've done some other changes I'm into (mostly the last bugfixes for the next 3.1.x and I've just found another bug in pool, not released, but the maintenance branch will need some attention.

I think this MR is good to merge, thank you very much!